README.md

```diff
@@ -1,13 +1,90 @@
-# Hang In There
+# Hang in there!
+#### a fun app to generate random motivational posters, and make your own!
 
-A boilerplate repo. 
+![image or gif](link in the parens) - I will work on this in the morning!
 
-## Set Up
+[Make your own poster!](https://pcmueller.github.io/hang-in-there-boilerplate/)
 
-1. One teammate: fork this repository
-2. Go to settings and turn on GitHub Pages for this repository
-3. All teammates: clone down this repository
-4. `cd` into the repository
-5. Run `open index.html` to view it in the browser
+#### Overview & [Project](https://frontend.turing.io/projects/module-1/hang-in-there.html) Goals
+- Sometimes you need a pick me up. Remember those motivational posters that were all over the place in classrooms and online? We’re going to make our own!
+- Write clean, DRY JavaScript to store our data
+- Use a provided class by creating object instances using the new keyword
+- Manipulate the page after it has loaded adding, removing, and updating elements on the DOM
+- Begin to understand the connection between HTML, CSS and JavaScript
 
-Project spec & rubric can be found [here](https://frontend.turing.io/projects/module-1/hang-in-there.html)
+#### Learning Goals
+- Develop a better understanding of learned concepts
```

Comment:
Later in the doc we have sentences ending with a period.  For consistency in style I think we should choose one or the other!

Reply:
√